cdserver/features/shelters/providers_index.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Providers Index
+  As a capdash user,
+  I want to see the list of providers
+  so that I can see their administrative and billing information.
+
+@ignore
+Scenario: list all providers
+  Given Providers in the system
+  When I navigate to /providers
+  Then I should see a list of all providers
```

Comment:
this should be "I should see the following provider information: "  then a table of information, including computed columns if any